### PR TITLE
WebUI: use Cockpit's run-tests

### PR DESCRIPTION
**Pull request by jelly**
The new run-tests version determines what machine_class to use from the to be run tests as long as machine_class is set correctly for all tests it will pick up VirtInstallMachine.

Requires https://github.com/cockpit-project/cockpit/pull/18627 to be merged first.

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me - a very nice improvement, thanks a lot! :)

**Review by VladimirSlavik — APPROVED** [review]
Thanks!

File: ui/webui/Makefile.am [hunk 1]
```diff
@@ -71,7 +71,7 @@ COCKPIT_REPO_FILES = \
 	$(NULL)
 
 COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
-COCKPIT_REPO_COMMIT = 54a8ebd2dfe4d53bcfa5db660701469e3f21dbba # 288.2 + cockpit-po-plugin fix
+COCKPIT_REPO_COMMIT = 84c7ee507b57485b5edd97ed3321982a59032f20 # 289 + run-tests machine_class fix
```
Comment (VladimirSlavik) [inline ui/webui/Makefile.am:74]: Just a random though, unrelated to the PR itself - perhaps this could be a separate file, same as `branch-config.mk` ?

**Pull request merged**